libs/sdk-ui-tests/stories/visual-regression/web-components/insightStories.tsx

```diff
@@ -0,0 +1,51 @@
+// (C) 2022 GoodData Corporation
+import React from "react";
+import { ReferenceRecordings } from "@gooddata/reference-workspace";
+import { storiesOf } from "../../_infra/storyRepository";
+import { WebComponents } from "../../_infra/storyGroups";
+import "../../_infra/webComponents";
+
+const insightIds = [
+    ReferenceRecordings.Insights.PivotTable.TwoMeasuresAndGrandTotalsAndMultipleSubtotals,
+    ReferenceRecordings.Insights.AreaChart.SingleMeasureWithViewBy,
+    ReferenceRecordings.Insights.BarChart.SingleMeasure,
+    ReferenceRecordings.Insights.BubbleChart.XAndYAxisMeasuresWithViewBy,
+    ReferenceRecordings.Insights.Xirr.CorrectConfig,
+    ReferenceRecordings.Insights.BulletChart.PrimaryTargetAndComparativeMeasures,
+    ReferenceRecordings.Insights.ColumnChart.SingleMeasureWithViewBy,
+    ReferenceRecordings.Insights.ComboChart.MultiplePrimaryAndSecondaryMeasuresWithViewBy,
+    ReferenceRecordings.Insights.DonutChart.SingleMeasureWithViewBy,
+    ReferenceRecordings.Insights.FunnelChart.SingleMeasureWithViewBy,
+    ReferenceRecordings.Insights.Headline.TwoMeasures,
+    ReferenceRecordings.Insights.Heatmap.MeasureRowsAndColumns,
+    ReferenceRecordings.Insights.LineChart.TwoMeasuresWithTrendBy,
+    ReferenceRecordings.Insights.PieChart.TwoMeasures,
+    ReferenceRecordings.Insights.ScatterPlot.XAxisMeasureAndAttribute,
+    ReferenceRecordings.Insights.Treemap.TwoMeasuresAndViewBy,
+].map((insight) => insight.obj.insight.identifier);
+
+type InsightContainerProps = {
+    insight: string;
+    locale?: string;
+    title?: string;
+};
+
+const InsightContainer: React.FC<InsightContainerProps> = (props) => {
+    return <gd-insight style={{ height: 500 }} {...props} />;
+};
+
+storiesOf(`${WebComponents}/Insight`)
```

Comment:
I can't run screenshots on local machine due to M1 mac. Would appreciate your help in getting screenshots for this snapshots.

Reply:
Unfortunately, I also have M1 :-/
@kandl can you please help?